### ISSUE TITLE
PmdAssert.java:78-79: Add Cactoos dependency to...

### ISSUE THREAD

**Issue by 0pdd** [teamed/qulice#1009]
The puzzle `966-537ebfc3` from #966 has to be resolved: 

https://github.com/teamed/qulice/blob/48e372b632e8a2f4092c5ad94a1501f4d04e24a1/qulice-pmd/src/test/java/com/qulice/pmd/PmdAssert.java#L78-L79

The puzzle was created by Nico Korthout on 21-Jan-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1009](https://github.com/teamed/qulice/issues/1009) is now in scope, role is `DEV`

**Comment by krzyk** [comment]
@0crat assign @korthout

**Comment by krzyk** [comment]
@korthout Let me know if you don't want to work on this, also if you find any issue that you would like to work on let me know I can assign it to you, you'll get some points to leave the sandbox.

**Comment by 0crat** [comment]
> @0crat assign @korthout ([here](https://github.com/teamed/qulice/issues/1009#issuecomment-458921049))

@krzyk The job [#1009](https://github.com/teamed/qulice/issues/1009) assigned to @korthout[/z](https://www.0crat.com/u/korthout), here is [why](https://www.0crat.com/footprint/C3T49A35L/56fb3b5b-1675-460c-a32e-63b3ab44e97a); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); @korthout[/z](https://www.0crat.com/u/korthout) is not a member of this project yet, but they can request to join, as [§1](http://www.zerocracy.com/policy.html#1) explains; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by korthout** [comment]
@krzyk I can work on this. However, in another project I'm waiting on #878, which in turn is waiting on #959. So it would be good for me to actively work on that too. It might take me some extra time because I don't yet have all the know-how on working with SpotBugs to finish that job.

[Yegor said](https://www.yegor256.com/2015/02/16/it-is-not-a-school.html):
> It's not a school

But if you don't mind I just take some extra time in finishing that job. It just takes more time for the same points.

**Comment by krzyk** [comment]
@korthout OK, so I'll assign you to #959 first

**Comment by korthout** [comment]
@krzyk thanks

**Comment by 0pdd** [comment]
The puzzle `966-537ebfc3` has disappeared from the source code, that's why I closed this issue.

**Comment by krzyk** [comment]
@0crat help

**Comment by 0crat** [comment]
Job [#1009](https://github.com/teamed/qulice/issues/1009) is not in the agenda of @korthout[/z](https://www.0crat.com/u/korthout), can't retrieve data and time of add

**Comment by 0crat** [comment]
> @0crat help ([here](https://github.com/teamed/qulice/issues/1009#issuecomment-487955548))

@krzyk Can't understand "help", try one of these:
  - `assign` Assign a performer to this issue
  - `boost` Set a boost for the job
  - `continue` Remove a job's impediment
  - `hello` Just say hello
  - `in` Register this issue in scope (WBS)
  - `out` Close the order and remove this job from scope
  - `pay` Pay a user some extra cash
  - `quality` Review a task
  - `resign` Resign from this issue
  - `status` Check the status of the job
  - `version` Print current version of the bot
  - `wait` Register an impediment for a job

**Comment by krzyk** [comment]
@0crat pay @zCRUSADERz 15min

**Comment by 0crat** [comment]
The job [#1009](https://github.com/teamed/qulice/issues/1009) is now out of scope

**Comment by 0crat** [comment]
> @0crat pay @zCRUSADERz 15min ([here](https://github.com/teamed/qulice/issues/1009#issuecomment-487958882))

@krzyk Argument "@zCRUSADERz" doesn't match regex `\d+min`

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @korthout[/z](https://www.0crat.com/u/korthout)
